columnflow/columnar_util.py

```diff
@@ -2529,9 +2529,7 @@ def __init__(
         self: DaskArrayReader,
         path: str,
         open_options: dict | None = None,
-        # TODO: we want to go back to SLICES as soon as possible, but currently there is an issue
-        #       that we load all events into memory for each chunk when using SLICES
-        materialization_strategy: MaterializationStrategy = MaterializationStrategy.PARTITIONS,
+        materialization_strategy: MaterializationStrategy = MaterializationStrategy.SLICES,
```

Comment:
`dak_array[start:end]` seems to be working now without memory leak, so back to slicing.